src/Symfony/Component/Process/Process.php

```diff
@@ -1220,6 +1223,36 @@ public function getStartTime(): float
         return $this->starttime;
     }
 
+    /**
+     * setWinOptions sets windows proc_open specific options
+     * useful with the option create_new_console to open a process and an detach it
+     * from your php script so it can run in background
+     * requires minimum PHP 7.4.4.
+     */
+    public function setWinOptions(array $options)
+    {
+        if ('\\' !== \DIRECTORY_SEPARATOR) {
```

Comment:
I really think we can remove this check and name the method `setOptions()`
unsupported options will be a no-op on Linux. That's most likely fine and if devs care, they will have to handle the specific portability needs anyway.

Reply:
Ok, I changed  the name and remove the check, my initial intention was it to make it more simple to understand as all the options of `proc_open` are windows only and there is a very high probability that this will stay the same in the future.